src/common/Popups.js

```diff
@@ -26,26 +27,66 @@ class Popups extends Component {
       optionValue: nextProps.optionValue
     });
   }
-
+ 
   handleClose = () => {
     this.setState({show : false });
   }
-
+ 
   handleSubmit = (e) => {
     e.preventDefault();
     // send request to server to update the data 
     // ADD CODE HERE
   }
-
+ 
   handleDeactivateAccount = () => {
     console.log("Deactivating account...");
     // send request to the server to deactivate account 
     // ADD CODE HERE
   }
-
+ 
   onChange = (e) => {
     this.setState({[e.target.name] : e.target.value });
   }
+  handleChange = event => {
```

Comment:
@tanujvyas10 This function is not used anywhere, this is already handled by onChange function, please remove this.

Reply:
sure , i didnot saw that, i will correct this asap